tests/test_pm.py

```diff
@@ -164,6 +175,9 @@ def create_policy(connector, zone, policy_spec=None, policy=None, defaults=None)
     return resp
 
 
+POlICY_DOMAINS = ['vfidev.com', 'vfidev.net', 'venafi.example']
```

Comment:
@rvelaVenafi is there a reason for the lower case "l" here? i.e. why is it not `POLICY_DOMAINS`?

Reply:
Of course there is! A big big typo!
Fixing it asap.

Reply:
Done!